solr/solr-ref-guide/modules/getting-started/pages/introduction.adoc

```diff
@@ -16,28 +16,32 @@
+Since Solr is Apache 2.0 licensed open source software designed for extensibility, it gives you the freedom to adapt or optimize it for almost any commercial or non-commercial use case.
 
-Solr is based on open standards and it is highly extensible.
-Solr queries are simple HTTP request URLs and the response is a structured document: mainly JSON, but it could also be XML, CSV, or other formats.
-This means that a wide variety of clients will be able to use Solr, from other web applications to browser clients, rich client applications, and mobile devices.
-Any platform capable of HTTP can talk to Solr.
-See xref:deployment-guide:client-apis.adoc[] for details on client APIs.
+Solr's xref:query-guide:query-syntax-and-parsers.adoc[query syntax and parsers] offer support for everything from the simplest keyword searching through to complex queries on multiple fields and xref:query-guide:faceting.adoc[faceted] search results.
+xref:query-guide:collapse-and-expand-results.adoc[Collapsing] and xref:query-guide:result-clustering.adoc[clustering] results offer compelling features for e-commerce and storefronts.
+Streaming calculations across large sets of text documents are available via xref:query-guide:streaming-expressions.adoc[streaming expressions].
+Powerful xref:query-guide:math-expressions.adoc[math expressions] provide the backbone for advanced analysis and predictive analytics use cases.
 
-Flexible schema configurations allow nearly any type of data to be stored in Solr.
-The xref:indexing-guide:schema-elements.adoc[] has more details on these options.
+Advanced relevancy tuning is also supported.
+Solr provides access to almost all of Lucene's text analysis features including tokenization, stemming, synonyms and much more, allowing you to tune relevancy based on knowledge of your users and your domain.
+Solr even allows for customization of relevancy via machine learning using the xref:query-guide:learning-to-rank.adoc[] feature.
 
-Solr offers support for the simplest keyword searching through to complex queries on multiple fields and faceted search results.
-Collapsing and clustering results offer compelling features for e-commerce and storefronts.
-Powerful math expressions provide the backbone for advanced analytics use cases.
-The xref:query-guide:query-syntax-and-parsers.adoc[] has more information about searching and queries.
+Queries are transmitted to Solr via HTTP 1.0, 1.1 or 2.0 requests and the response is typically a list of structured document descriptors.
+In the classic example, 10 descriptors are returned, each including a URL to locate the document (often rendered as "10 blue links"). However, Solr can go far beyond document locators and many other types of document metadata might also be included.  Flexible schema configurations allow nearly any type of metadata to be associated with a document indexed in Solr.
```

Comment:
descriptors is maybe an less common choice versus "result" or "document"...

Reply:
Although "document" is the traditional term, it is quite misleading if you aren't familiar with search. A lay person hearing that a "document" is returned would think that the entire set of bits that went in are coming back out. While that could be achieved and in rare cases is what's desired, it's abnormal and in many cases it is bad design, so I specifically was careful not to use "document" as a noun here. In most cases what we return is a much shorter "pointer plus metadata". Descriptor seemed apropos but if you can think of a more elegant word that clearly communicates the same thing to a reader that has never implemented search, I'm all ears.

Reply:
That distinction between result document and indexed document can be clarified later of course, I wouldn't propose changing terminology everywhere, but here on the very front of everything in a blurb too short for definitions, it's going to leave the reader with a false concept.